app/src/main/java/com/example/mycleaninglog/MainActivity.kt

```diff
@@ -27,11 +27,7 @@ class MainActivity : ComponentActivity() {
                             .background(MaterialTheme.colors.background)
                             .padding(24.dp)
                     ) {
-                        //calls the 1st level expandable cards. hardcoded because they should never change. These make up the main menu
-                            ExpandableCard(title = "Rooms")
-                            ExpandableCard(title = "Common Tasks")
-                            ExpandableCard(title = "Upcoming Tasks")
-
+                        HomeScreen()
```

Comment:
Made this HomeScreen call because its more efficient and cleaner then calling three ExpandableCard functions